Source/utils/paths.cpp

```diff
@@ -37,7 +37,7 @@ void AddTrailingSlash(std::string *path)
 
 std::string *FromSDL(char *s)

```

Comment:
Why does this function even return a std::string* instead of just an std::string?

Reply:
I think @glebm did it to avoid an issue with GCC, can't find the comment, don't know if this is still relevant.
Not part of the PR though...